lametro/templates/lametro/person.html

```diff
@@ -72,7 +76,7 @@ <h4>
         <div class="col-sm-4 mobile-only">
             <div class="row">
                 <div class="col-xs-4">
-                    <img src='{{person.headshot_url}}' alt='{{person.name}}' title='{{person.name}}' class='img-responsive img-thumbnail' />
+                    <img src='{% if person.image %}{{person.image}}{% else %}{{person.headshot_url}}{% endif %}' alt='{{person.name}}' title='{{person.name}}' class='img-responsive img-thumbnail' />
```

Comment:
How about updating `person.headshot_url` to do this check, so you don't have to do this conditional in the template?